io-package.json

```diff
@@ -724,5 +724,231 @@
+                        "it": "Si è verificato un problema durante l'accesso ai file sul tuo sistema. Puoi provare a eseguire il programma di riparazione dell'installazione.",
+                        "es": "Hay un problema al acceder a los archivos de su sistema. Puede intentar ejecutar el reparador de instalación.",
+                        "pl": "Wystąpił problem z dostępem do plików w systemie. Możesz spróbować uruchomić program do naprawy instalacji.",
+                        "zh-cn": "访问系统上的文件时出现问题。您可以尝试运行安装修复程序。"
+                    },
+                    "regex": [
+                        "^Exception-Code: ENOSPC"
+                    ],
+                    "limit": 10
+                },
+                {
+                    "category": "nonExistingFileErrors",
+                    "name": {
+                        "en": "Issues with non-existing file or directory",
+                        "de": "Probleme mit nicht vorhandener Datei oder nicht vorhandenem Verzeichnis",
+                        "ru": "Проблемы с несуществующим файлом или каталогом",
+                        "pt": "Problemas com arquivo ou diretório não existente",
+                        "nl": "Problemen met niet-bestaand bestand of directory",
+                        "fr": "Problèmes avec un fichier ou un répertoire inexistant",
+                        "it": "Problemi con file o directory inesistenti",
+                        "es": "Problemas con archivos o directorios que no existen",
+                        "pl": "Problemy z nieistniejącym plikiem lub katalogiem",
+                        "zh-cn": "文件或目录不存在的问题"
+                    },
+                    "severity": "alert",
+                    "description": {
+                        "en": "A process tries to access a non-existing file or directory.",
+                        "de": "Ein Prozess versucht, auf eine nicht vorhandene Datei oder ein nicht vorhandenes Verzeichnis zuzugreifen.",
+                        "ru": "Процесс пытается получить доступ к несуществующему файлу или каталогу.",
+                        "pt": "Um processo tenta acessar um arquivo ou diretório não existente.",
+                        "nl": "Een proces probeert toegang te krijgen tot een niet-bestaand bestand of map.",
+                        "fr": "Un processus tente d'accéder à un fichier ou un répertoire inexistant.",
+                        "it": "Un processo tenta di accedere a un file o una directory inesistente.",
+                        "es": "Un proceso intenta acceder a un archivo o directorio que no existe.",
+                        "pl": "Proces próbuje uzyskać dostęp do nieistniejącego pliku lub katalogu.",
+                        "zh-cn": "进程尝试访问不存在的文件或目录。"
+                    },
+                    "regex": [
+                        "^Exception-Code: EHOSTDOWN"
```

Comment:
Text passt nicht ganz oder ?! EHOSTDOWN sollten netzwerk.filesyssteme sein (nfs oder sowas)

Reply:
ENOENT wäre nicht vorhanden :-)

Reply:
And again